doc/tools/index.rst

```diff
@@ -0,0 +1,112 @@
+DAGMC Tools
+======
+
+The DAGMC infrastructure has a number of tools that can help with issues that
+arise when using the toolkit. This document serves to better inform and 
```

Comment:
I don't think it is just "issues that arise." For example, running `dagmc_preproc` is a mandatory step of the workflow.